examples/elixir_euler_potential_temperature_baroclinic_instability.jl

```diff
@@ -0,0 +1,278 @@
+# An idealized baroclinic instability test case
+# For optimal results consider increasing the resolution to 16x16x8 trees per cube face.
+#
```

Comment:
Can you please give a rough estimate of how long this will run (for some number of threads you used)?